agent/exec/container/container.go

```diff
@@ -12,6 +13,7 @@ import (
 	"github.com/docker/docker/api/types/events"
 	"github.com/docker/docker/api/types/filters"
 	"github.com/docker/docker/api/types/network"
+	"github.com/docker/go-connections/nat"
```

Comment:
Which has nothing to do with connections or nat...

Sigh.

Outside the scope of this PR, so please ignore.